pkg/cmd/cli/cmd/status.go

```diff
@@ -160,7 +160,9 @@ func (o StatusOptions) RunStatus() error {
 		if err != nil {
 			return err
 		}
-		data, err := dot.Marshal(g, o.namespace, "", "  ", false)
+		// DOT graph names cannot contain dashes unless it is a double-quoted string
+		// See http://www.graphviz.org/doc/info/lang.html
+		data, err := dot.Marshal(g, fmt.Sprintf("%q", o.namespace), "", "  ", false)
```

Comment:
the upstream API is ambiguous about which type of ID the `name` arg is (string or HTML) that it is not possible to know whether a value like `<x>y</x>` was intended as a literal string or as an HTML string. I'd settle for a `quote()` helper func that quotes the value as a literal, and passes the result to the existing `Marshal` func